examples/app-npm-webpack/webpack-config.js

```diff
@@ -0,0 +1,55 @@
+var webpack = require( "webpack" );
+var CommonsChunkPlugin = require( "webpack/lib/optimize/CommonsChunkPlugin" );
+var HtmlWebpackPlugin = require( "html-webpack-plugin" );
+var globalizePlugin = require( "globalize-webpack-plugin" );
```

Comment:
`GlobalizePlugin` (uppercase) like the other plugins: `CommonsChunkPlugin`, `HtmlWebpackPlugin`.